mis_builder/models/mis_safe_eval.py

```diff
@@ -22,7 +23,10 @@ def mis_safe_eval(expr, locals_dict):
     """
     try:
         c = test_expr(expr, _SAFE_OPCODES, mode='eval')
-        globals_dict = {'__builtins__': _BUILTINS}
+        builtins = _BUILTINS.copy()
+        builtins.update({
+            'datetime':datetime})
+        globals_dict = {'__builtins__': builtins}
```

Comment:
It should not be necessary to fiddle with this low level code. Would it work to add this in prepare_locals_dict in mis_report?